dashboard/modules/tests/test_head.py

```diff
@@ -109,6 +110,27 @@ async def test_file(self, req) -> aiohttp.web.FileResponse:
         logger.info("test file: %s", file_path)
         return aiohttp.web.FileResponse(file_path)
 
+    async def blocking_async_func(self, seconds):
+        import time
+
+        time.sleep(seconds)
+        return seconds
+
+    @routes.get("/test/block_event_loop")
+    async def block_event_loop(self, req) -> aiohttp.web.Response:
+        """
+        Simulates a blocked event loop. To be used for testing purposes only. Creates a
+        task that blocks the event loop for a specified number of seconds.
+        """
+        seconds = float(req.query.get("seconds", 0.0))
+        t = asyncio.get_event_loop().create_task(self.blocking_async_func(seconds))
+        await t
```

Comment:
can we just time.sleep here?

Reply:
done. This was intended for the other aiodebug feature to detech blocking tasks. Removed (to time.sleep) for now.